src/components/UsersList/UsersList.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from '../User/User';
+import { TypeUser } from '../../types';
+
+export const UsersList = ({ users, handle, selectedUserId }) => (
+  <select
+    name="selectedUserId"
+    value={selectedUserId}
+    onChange={handle}
+  >
+    <option value="">
+      Choose a user
+    </option>
+
+    {users.map(user => (
+      <User {...user} key={user.id} />
```

Comment:
it's a bad idea to move select's option to the separate component, let's move it back here